packages/config/config/devices/0x0063/templates/jasco_template.json

```diff
@@ -395,5 +395,72 @@
 				"value": 2
 			}
 		]
+	},
+	"3way_setup": {
+		"label": "3-Way Setup",
+		"description": "Set the 3-way switch configuration as add-on or standard.",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Add-on",
+				"value": 0
+			},
+			{
+				"label": "Standard",
+				"value": 1
+			}
+		]
+	},
+	"led_light_color": {
+		"label": "LED Light Color",
```

Comment:
We mostly use `LED Indicator`, not `LED Light` to refer to built-in LEDs. Please rename for consistency